cs/tech.go

```diff
@@ -415,13 +418,16 @@ func (t *Tech) GetPlayerCost(techLevels TechLevel, spec MiniaturizationSpec, cos
 		cost = cost.MultiplyFloat64(1 + costOffset.Bomb)
 	case TechCategoryTorpedo:
 		cost = cost.MultiplyFloat64(1 + costOffset.Torpedo)
+		//case TechCategoryOrbital && {some way to check if item is gate}
+		//	cost = cost.MultiplyFloat64(1 + costOffset.Stargate)
```

Comment:
was this left in by mistake?

Reply:
Nah - I just wanted to put a comment there to remind myself to fix thing once the tag system gets implemented. 
Probably needs a big orange "TODO" marker come to think about it...